clojure-mode.el

```diff
@@ -584,7 +583,14 @@ If JUSTIFY is non-nil, justify as well as fill the paragraph."
              (concat paragraph-separate "\\|\\s-*\".*[,\\.]$"))
             (fill-column (or clojure-docstring-fill-column fill-column))
             (fill-prefix (clojure-docstring-fill-prefix)))
-        (fill-paragraph justify))
+        (let* ((beg (nth 8 (syntax-ppss)))
```

Comment:
I'd add some comment here explaining what this code does. Even I forget from time to time the behavior of functions like `syntax-ppss`, which is pretty easy with a name like this and absurd return values.